security/tls-1.3.md

```diff
@@ -52,11 +52,13 @@ TLS version is not configurable with flags.
 For Ironic, it should be noted that:
 
 - Ironic external endpoints are secured, while pod-internal traffic is not.
```

Comment:
Should this also be changed? If we don't use `http` for pod internal traffic, but sockets?